src/Adapter/AbstractObjectModelProvider.php

```diff
@@ -64,4 +64,27 @@ protected function getObjectModel(int $id, string $objectModelClass, string $exc
 
         return $objectModel;
     }
+
+    /**
+     * @param int $id
+     * @param string $objectModelName
+     * @param string $exceptionClass
+     * @param int $errorCode
+     *
+     * @throws CoreException
+     */
+    protected function assertObjectModelExists(int $id, string $objectModelName, string $exceptionClass, int $errorCode = 0): void
```

Comment:
```suggestion
    protected function assertObjectModelExists(int $id, string $objectTableName, string $exceptionClass, int $errorCode = 0): void
```

We need the table name here, so it's easier to understand if the parameter expresses this

Reply:
> But some behat tests should be added though

I've tried introducing behats for attachments, but failed.
Add/Update Attachment handler uses move_uploaded_file inside, which must be uploaded by HTTP POST, so there is no way (at least I don't know any) to test it

Reply:
I believe you can drop this one, since it's in the Repository abstract class and it was not used anywhere else